File: CPP/SumArrays/MySolution.cpp
#include <vector>
#include <numeric>

int sum(std::vector<int> nums) {
  return std::accumulate(nums.begin(), nums.end(), 0);
}

// Original kata: https://www.codewars.com/kata/53dc54212259ed3d4f00071c
// My solution: https://www.codewars.com/kata/reviews/5d4dd682dec905000197262d/groups/5d4e59045abb070001b0c82c

File: Python/DoesMyNumberLookBigInThis/MySolution.py
def narcissistic(value):
    return sum(int(digit) ** len(str(value)) for digit in str(value)) == value

# Original kata: https://www.codewars.com/kata/5287e858c6b5a9678200083c/train/python
# My solution: https://www.codewars.com/kata/reviews/55db4cb62ede87b90600007e/groups/5800a63715e97eb150000674